python_modules/dagster/dagster_tests/storage_tests/test_instigation_logger.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from dagster._core.definitions.instigation_logger import InstigationLogger
+
+
+def test_gets_correct_logger(caplog):
+    custom_logger_name = "foo"
+    caplog.set_level(logging.INFO, logger=custom_logger_name)
```

Comment:
trying to find the magic incantation that will get caplog to capture the logs emitted by "foo" later in this test. Unable to figure it out right now....